src/pages/academy/components/_pagination.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
```

Comment:
This might be used in other pages outside academy in the future. Can we make it more generic